src/rai/rai/cli/rai_cli.py

```diff
@@ -35,42 +41,80 @@ def parse_whoami_package():
         "documentation_root", type=str, help="Path to the root of the documentation"
     )
     parser.add_argument(
-        "output",
+        "--output",
         type=str,
+        required=False,
+        default=None,
         help="Path to the output directory",
     )
     args = parser.parse_args()
+    save_dir = args.output if args.output is not None else args.documentation_root
+
+    def build_docs_vector_store():
+        logger.info("Building the robot docs vector store...")
+        faiss_index = FAISS.from_documents(docs, OpenAIEmbeddings())
+        faiss_index.add_documents(docs)
+        faiss_index.save_local(save_dir)
+
+    def build_robot_identity():
+        logger.info("Building the robot identity...")
+        prompt = (
+            "You will be given a robot's documentation. "
+            "Your task is to identify the robot's identity. "
+            "The description should cover the most important aspects of the robot with respect to human interaction, "
+            "as well as the robot's capabilities and limitations including sensor and actuator information. "
+            "If there are any images provided, make sure to take them into account by thoroughly analyzing them. "
+            "Your description should be thorough and detailed."
+            "Your reply should start with I am a ..."
+        )
+        llm = ChatOpenAI(model="gpt-4o-mini")
```

Comment:
Model selection should be added to some global RAI/langchain config - it will become hell for users without OpenAI key to use rai

Reply:
Done